templates/CRM/Contact/Import/Form/DataSource.tpl

```diff
@@ -83,7 +83,7 @@
          </tr>
         { /if}
 
-        {if $form.disableUSPS}
+        {if isset($form.disableUSPS)}
```

Comment:
Is this expected to be a boolean? Then isset might always be true even if false/0.

Reply:
Oh nevermind.